pkg/api/vlabs/validate_test.go

```diff
@@ -598,7 +598,7 @@ func Test_KubernetesConfig_Validate(t *testing.T) {
 			ProxyMode:     "iptables",
 		}
 
-		if err := c.Validate(k8sVersion, false, true, false); err == nil && !common.IsKubernetesVersionGe(k8sVersion, "1.18.0-alpha.2") {
+		if err := c.Validate(k8sVersion, false, true, false); err == nil && !common.IsKubernetesVersionGe(k8sVersion, "1.18.0") {
```

Comment:
@aramase I assume the specific alpha version is intentional here. We usually remove code references to pre-release versions when the final ships, but if you think it's useful to be specific about which alpha version introduced support, I can revert this.

Reply:
@mboersma I used alpha.2 intentionally because this was not available in alpha.1. But we can now switch to using `1.18.0`.